### PR TITLE
Update to Angular 13

### PR DESCRIPTION
**Pull request by iangabrielsanchez**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -28,18 +28,6 @@
...
-          "builder": "@angular-devkit/build-angular:tslint",
-          "options": {
-            "tsConfig": [
-              "projects/angular-formio/tsconfig.lib.json",
-              "projects/angular-formio/tsconfig.spec.json"
-            ],
-            "exclude": [
-              "**/node_modules/**"
-            ]
-          }
```
Comment (iangabrielsanchez) [inline angular.json:42]: > Deprecated @angular-devkit/build-angular:tslint builder has been removed. Use angular-eslint/angular-eslint instead.

source: https://github.com/angular/angular-cli/blob/master/CHANGELOG.md#angular-devkitbuild-angular-4

File: projects/angular-formio/auth/package.json [hunk 2]
```diff
@@ -1,10 +1,6 @@
 {
   "ngPackage": {
     "lib": {
-      "umdModuleIds": {
-        "lodash": "_",
-        "formiojs": "formiojs"
-      }
```
Comment (iangabrielsanchez) [inline projects/angular-formio/auth/package.json:7]: >* We no longer generate UMD bundles. The below options which were used for UMD bundle generation has also been removed;
>   - `umdModuleIds`
>   - `amdId`
>   - `umdId`

Source: https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md#-breaking-changes

**Comment by travist** [comment]
I noticed that there is this PR. https://github.com/formio/angular/pull/835

Is there any reason why this one is not good enough?

**Comment by iangabrielsanchez** [comment]
> I noticed that there is this PR. #835
> 
> Is there any reason why this one is not good enough?

You're right, I didn't notice. 😅 closing my pr now

**Pull request closed without merging**